### PR TITLE
Update to go 1.14.7

### LINKED ISSUE #174: Ubuntu 20.04 crash with mlock limit
On ubuntu 20.04 the latest version keeps crashing with

> runtime: mlock of signal stack failed: 12
> runtime: increase the mlock limit (ulimit -l) or
> runtime: update your kernel to 5.3.15+, 5.4.2+, or 5.5+
> fatal error: mlock failed
>

### PR DESCRIPTION
**Pull request by lucaslorentz**
Try to fix #174

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ jobs:
   steps:
   - task: GoTool@0
     inputs:
-      version: '1.14'
+      version: '1.14.7'
```
Comment (francislavoie) [inline azure-pipelines.yml:9]: Does that azure task support `1.14.x`? That's what we use for github actions. Strange if not...

Comment (lucaslorentz) [reply azure-pipelines.yml:9]: I didn't find that in the docs.
https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/tool/go-tool?view=azure-devops
But might work, I will try

Comment (francislavoie) [reply azure-pipelines.yml]: Or if it supports semver patterns... `^1.14.7` :man_shrugging:

Comment (lucaslorentz) [reply azure-pipelines.yml]: 1.14.x worked

**Pull request merged**